packages/core/src/parsers/linkProcessor.js

```diff
@@ -45,6 +45,19 @@ function convertRelativeLinks(node, cwf, rootPath, baseUrl) {
   node.attribs[linkAttribName] = path.posix.join(baseUrl || '/', resourcePathFromRoot);
 }
 
+function convertMdExtToHtmlExt(node) {
+  if (node.attribs && node.attribs.href) {
```

Comment:
should we limit this to anchor tags as well? `node.name === 'a'`, since some other tags may use `href` for different purposes and have unintended side effects

Reply:
agreed! thanks for bringing this up!